### PR TITLE
Added pixel type attribute to the output image of program shadow. Fixes #5187

### PR DESCRIPTION
**Pull request by kaitlyndlee**
shadow would sometimes output an image with all LRS pixels if the input DEM was not a 32-bit image. When adding the pixel type attribute to the output image, this fixed the problem. 

## Description

Added pixel type attribute to the output image and set it to real so that the output image did not contain all LRS pixels.

## Related Issue

https://isis.astrogeology.usgs.gov/fixit/issues/5187

## Motivation and Context

Output images are now correct.

## How Has This Been Tested?

Tested changes with a 16-bit image that would reproduce the problem. After changes, the output DEM did not contain all LRS pixels.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/base/apps/shadow/shadow.xml [hunk 1]
```diff
@@ -2,26 +2,28 @@
...
-    This program will create a shaded-relief cube from a DEM and match cube.<br/><br/>
+    This program, shadow, will create a shaded-relief cube from a digital
+    elevation model (DEM) and a MATCH cube.<br/><br/>
 
-    This program operates much like 'shade' but instead of an azimuth/elevation input, we use the
-      sun's position at the center of the given cube or at a given time and, by default, factor in
-      shadows cast by features.<br/><br/>
+    We use the sun's position at the center of the input cube or a user-defined
+    observation time. By default, we factor in shadows cast by features.
+    This program operates much like the ISIS3 'shade' program which requires azimuth/elevation as input.<br/><br/>
```
Comment (SgStapleton) [inline isis/src/base/apps/shadow/shadow.xml]: "...which instead requires..." maybe to specifically delineate the difference between the two apps.

File: isis/src/base/apps/shadow/shadow.xml [hunk 2]
```diff
@@ -170,6 +173,7 @@
       </parameter>
       <parameter name="TO">
         <type>cube</type>
+        <pixelType>real</pixelType>
```
Comment (SgStapleton) [inline isis/src/base/apps/shadow/shadow.xml:176]: This is really the only required change for this?

Comment (kaitlyndlee) [reply isis/src/base/apps/shadow/shadow.xml:176]: Yes, the change was recommended by the author of the ticket and fixed the bug.

**Review by SgStapleton — APPROVED** [review]
Looks good!

**Pull request merged**